services/src/main/java/org/exoplatform/addons/gamification/rest/AnnouncementRest.java

```diff
@@ -24,10 +28,16 @@
 
   private static final Log    LOG = ExoLogger.getLogger(AnnouncementRest.class);
 
+  private final CacheControl cacheControl;
+
+
   private AnnouncementService announcementService;
 
   public AnnouncementRest(AnnouncementService announcementService) {
     this.announcementService = announcementService;
+    this.cacheControl = new CacheControl();
```

Comment:
Are you using this ? why did you add it ?

Reply:
cleaned up!